### PR TITLE
Properly resolve full path for join in mapping

### PR DESCRIPTION
**Pull request by kshradhan**
| Q                       | A                                |
| ----------------------- | -------------------------------- |
| Fixed issues?           |  #424  |
| Tests added + passed?   | Yes                              |
| Changeset added?        | Yes                              |
| Documentation PR link   |                                  |
| Any dependency changes? | No                               |

### CODE REVIEW

File: packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_DatabaseTransformer.ts [hunk 1]
```diff
@@ -230,7 +229,7 @@ export const V1_transformRelationalOperationElement = (
     elementWithJoin.joins = operation.joinTreeNode
       ? extractLine(operation.joinTreeNode).map((node) => {
           const joinPtr = new V1_JoinPointer();
-          joinPtr.db = node.join.ownerReference.valueForSerialization ?? '';
+          joinPtr.db = node.join.ownerReference.value.path ?? '';
```
Comment (akphi) [inline packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_DatabaseTransformer.ts:232]:
```suggestion
          joinPtr.db = node.join.ownerReference.value.path;
```
```suggestion
          joinPtr.db = node.join.ownerReference.value.path ?? '';
```

File: .changeset/nervous-ants-laugh.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+---
+
+fix import statement see: [#424](https://github.com/finos/legend-studio/issues/424)
```
Comment (akphi) [inline .changeset/nervous-ants-laugh.md:5]:
```suggestion
Fix a problem with resolving full path for join ([#424](https://github.com/finos/legend-studio/issues/424))
```
```suggestion
fix import statement see: [#424](https://github.com/finos/legend-studio/issues/424)
```

File: packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_DatabaseTransformer.ts [hunk 3]
```diff
@@ -167,8 +167,7 @@ export const V1_transformTableAliasToTablePointer = (
   tableAlias: TableAlias,
 ): V1_TablePtr => {
   const tablePtr = new V1_TablePtr();
-  tablePtr.database =
-    tableAlias.relation.ownerReference.valueForSerialization ?? '';
+  tablePtr.database = tableAlias.relation.ownerReference.value.path ?? '';
```
Comment (akphi) [inline packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_DatabaseTransformer.ts:170]:
```suggestion
  tablePtr.database = tableAlias.relation.ownerReference.value.path;
```

**Comment by MauricioUyaguari** [comment]
closing in move for a more strategical fix to be done by @akphi

**Pull request closed without merging**